schematic/models/validate_attribute.py

```diff
@@ -30,6 +30,40 @@
 MessageLevelType = Literal["warning", "error"]
 ScopeTypes = Literal["set", "value"]
 
+ValueValidationOutput = tuple[pd.Series, pd.Series, pd.Series]
```

Comment:
Thoughts on moving this over to a dataclass similar to what you've done for `SetValidationOutput`. IMO a tuple can be a bit difficult to track the values since your comments you added below don't "move with" the object.

Reply:
I thought about it. If I were writing this from scratch I would have.

Reply:
Could you provide an estimate as to how long this would take and the downstream implications?

Reply:
Done!